src/Arcus.BackgroundJobs.DataBricks/DatabricksJobMetricsAdditionalOptions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GuardNet;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Arcus.BackgroundJobs.Databricks
+{
+    /// <summary>
+    /// Represents the additional options that the user can configure during the <see cref="IServiceCollectionExtensions.AddDatabricksJobMetrics"/> call.
+    /// </summary>
+    public class DatabricksJobMetricsAdditionalOptions
```

Comment:
Let's be consistent with https://github.com/arcus-azure/arcus.backgroundjobs/blob/master/src/Arcus.BackgroundJobs.CloudEvents/CloudEventBackgroundJobOptions.cs#L6 maybe?
```suggestion
    public class DatabricksJobMetricsJobOptions
```